packages/playground-server/src/middleware/create-account.ts

```diff
@@ -0,0 +1,83 @@
+import { Node } from "@counterfactual/node";
+import { Node as NodeTypes } from "@counterfactual/types";
+import { Context } from "koa";
+import { v4 as generateUUID } from "uuid";
+
+import {
+  ApiResponse,
+  CreateAccountRequest,
+  CreateAccountResponseData,
+  ErrorCode
+} from "../types";
+
+// Send { username, email, account_address=<web3_account>, signature=<signed msg by account> }
+
+function validateRequest(params: CreateAccountRequest): ApiResponse {
+  if (!params.username) {
+    return {
+      ok: false,
+      error: {
+        status: 400,
+        errorCode: ErrorCode.UsernameRequired
+      }
+    };
+  }
+
+  if (!params.email) {
+    return {
+      ok: false,
+      error: {
+        status: 400,
+        errorCode: ErrorCode.EmailRequired
+      }
+    };
+  }
+
+  if (!params.address) {
+    return {
+      ok: false,
+      error: {
+        status: 400,
+        errorCode: ErrorCode.AddressRequired
+      }
+    };
+  }
+
+  return { ok: true };
+}
+
+export default function createAccount(node: Node) {
+  return async (ctx: Context, next: () => Promise<void>) => {
+    const request = ctx.request.body as CreateAccountRequest;
+    const response = validateRequest(request);
+
+    if (!response.ok) {
+      ctx.body = response;
+      if (response.error) {
+        ctx.status = response.error.status;
+      }
+      return next();
+    }
+
+    const multisigResponse = await node.call(
```

Comment:
Is there a reason we don't use the method on the Channel object?

Reply:
@ebryn AFAIK, the intended usage of the Node API is through this method. Everything else is private.

Reply:
@cf19drofxots can you confirm?

Reply:
Yeah, it follows the same pattern as other method calls to the Node

https://github.com/counterfactual/monorepo/blob/master/packages/cf.js/API_REFERENCE.md#method-createmultisig